spec/models/movie_spec.rb

```diff
@@ -1,5 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe Movie, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "association" do
+    it { should have_many(:ratings).dependent(:destroy) }
+    it { should have_many(:shows).dependent(:destroy) }
+    it { should belong_to(:genre) }
+  end
+
+  describe "validation" do
```

Comment:
- Tìm hiểu về `describle, context, it`
- So sánh `let` vs `let!`

Reply:
describe dùng để nhóm các test case theo class, module, method, context dùng để nhóm theo điều kiện của các test case, it là 1 test case.
let chỉ được khởi tạo và tính toán ở trong 1 test case(trong 1 khối it) còn let! được khởi tạo và tính toàn từ trước và giá trị của nó được cache lại trước khi test case được thực thi.